kythe/cxx/indexer/cxx/KytheIndexerMain.cc

```diff
@@ -58,6 +58,7 @@ ABSL_FLAG(bool, use_compilation_corpus_as_default, false,
           "Use the CompilationUnit VName corpus as the default.");
 ABSL_FLAG(bool, experimental_record_dataflow_edges, false,
           "Emit experimental dataflow edges.");
+ABSL_FLAG(bool, use_abs_nodes, true, "Use abs nodes.");
```

Comment:
Should probably be `experimental_` especially since this is intended to be a temporary flag.